collect_app/build.gradle

```diff
@@ -107,7 +107,7 @@ allprojects {
 
 dependencies {
     compile fileTree(dir: 'libs', include: '*.jar')
-    compile 'com.google.android.gms:play-services:4.1.32'
+    compile 'com.google.android.gms:play-services-analytics:10.0.1'
```

Comment:
I've tested 
GeoPointMapActivity
GeoShapeGoogleMapActivity
GeoTraceGoogleMapActivity
with this form: [map.txt](https://github.com/opendatakit/collect/files/750382/map.txt)

GoogleDrive and GoogleSheets as well  - testing downloading forms from Gdrive, uploading to sheets and I havn't noticed any problems.